pkg/util/util.go

```diff
@@ -133,12 +133,12 @@ func RetryBackoffSize(dest string, size resource.Quantity, sizeFunc func(string,
 	err = sizeFunc(dest, size)
 	retryCount := 0
 	reduceSize := *size.Copy()
-	for err != nil && retryCount < 10 {
+	for err != nil && retryCount < 100 {
```

Comment:
Kind of not a fan of "if you don't succeed, try, try again."  Especially when retrying on any error and not something specific.

I also like being "more safe than sorry."   Think we're still cutting things a little too close here.  I would personally be inclined to just expand the image up to a certain threshold, like 90-95% of the disk.  Expect that the user gives us reasonably sized disks.  If there is extra space, help them out up to a point.  I'm afraid of shooting them in the foot later on.

Assuming you want to stay with this method, maybe an exponential backoff would be better?  Start with 25 or whatever, then double it after each failure?

Reply:
If we do exponential, we don't need this many tries. I am just trying to make sure we give the user the most amount of available space in the guest.

Reply:
I care more about avoiding failures and I personally think we should still be more conservative here.  But maybe @aglitke or others feel differently.